### DISCUSSION TITLE
no access to truenas

### DISCUSSION THREAD

**Discussion by shinzuka** [rern/rAudio#1675] [Fixed Issues]
Hello 
I don't have access to truenas which normally points to this path /truenas/music/

I configure it like this as usual on moodeaudio. Could it be different on rAudio?

![Capture d’écran du 2024-05-31 09-47-09](https://github.com/rern/rAudio/assets/68748242/54948e57-2772-4d88-9ad3-397f12dcb7ed)

![Capture d’écran du 2024-05-31 09-47-28](https://github.com/rern/rAudio/assets/68748242/34b04824-f78e-4da3-b0ea-c5f90436e54e)

**Comment by ha11oga11o** [comment]
Hello,

can you check or even manually edit /etc/fstab to look like this format?
This is in mine, it might help if you edit to suit your purpose?
Also, can you browse this in file explorer on pc?

`\\192.168.1.14\truenas\music`



//192.168.1.215/NasData  /mnt/MPD/NAS/NasData  cifs  noauto,username=ha11oga11o,password=xxxxxxxxxxxxxxxx,uid=45,gid=45,iocharset=utf8  0  0

Hope it helps.
